ios/BT/Info.plist

```diff
@@ -10,7 +10,7 @@
 	<key>CFBundleDevelopmentRegion</key>
 	<string>en</string>
 	<key>CFBundleDisplayName</key>
-	<string>PathCheck BT</string>
+	<string>PathCheck MN</string>
```

Comment:
Looks like this may have been included by accident?

Reply:
yep